.buildkite/test-pipeline.yaml

```diff
@@ -314,23 +314,24 @@ steps:
     - pytest -v -s models/test_oot_registration.py # it needs a clean process
     - pytest -v -s models/*.py --ignore=models/test_oot_registration.py
 
-- label: Decoder-only Language Models Test (Standard) # 35min
+- label: Decoder-only Language Models Test (Standard) # 14min
   #mirror_hardwares: [amd]
   source_file_dependencies:
   - vllm/
   - tests/models/decoder_only/language
   commands:
     - pytest -v -s models/decoder_only/language/test_models.py
 
-- label: Decoder-only Language Models Test (Extended) # 1h20min
-  nightly: true
+- label: Decoder-only Language Models Test (Extended %N) # 30min each
+  optional: true
```

Comment:
Temporary setting them to optional so I can run this inside the PR to test whether they're working or not.